### ISSUE TITLE
bbox in test set during evaluation

### ISSUE THREAD

**Issue by koalaaaaaaaaa** [woven-visionai/wts-dataset#1]
Hello, I'm wondering whether bbox groundtruth will be given in the test set during evaluation.
Thank you for your kindest reply.

**Comment by woven-visionai** [comment]
yes, we will provide the bbox (annotated + generated) for the test set as well, and user could use it as known information.
